fastlane/lib/fastlane/plugins/template/.rubocop.yml

```diff
@@ -0,0 +1,299 @@
+
```

Comment:
No, ideally we'd bundle the latest rubocop.yml with fastlane, so when the user run `fastlane new_plugin`, the latest config will be copied over to the new project. Does that make sense?